docs/usage.md

```diff
@@ -162,6 +190,61 @@ serviceRegistry:
 
 Execute the following command:
 
+```bash
+cnwan-reader servicedirectory --conf /path/to/configuration/file.yaml
+```
+
+or just:
+
+```bash
+cnwan-reader --conf /path/to/configuration/file.yaml
+```
+
+### With Cloud Map
+
+In the following example, the CNWAN Reader watches changes in
+AWS Cloud Map with the following requirements:
+
+* The *allowed* services have at least the `cnwan.io/traffic-profile`
+key in their metadata
+* The region is `us-west-2`
+* The credentials file is in `$HOME/.aws/credentials`, so there is no need to
+use the `--credentials-path` flag.
+* The endpoint of the adaptor is the default one
+(`http://localhost:80/cnwan/events`). In such a case there is no need to use
+the `--adaptor-api` flag, but here it is included for clarity.
+* Interval between two watches is `10 seconds`
+
+```bash
+cnwan-reader poll cloudmap \
+--region us-west-2 \
+--metadata-key cnwan.io/traffic-profile \
```

Comment:
Yes, that's because in future there is going to be the possibility of tracking more than one, although not enabled right now because servicedirectory still needs to be changed.

I havent posted any issues about this specifically because I'm going to include it in the one that re-organizes the cmd pkg

Reply:
That's ok, but the CLI examples should still use the plural form, because the code doesn't accept the singular form